backend/src/services/identity-access-token/identity-access-token-service.ts

```diff
@@ -39,7 +39,7 @@ export const identityAccessTokenServiceFactory = ({
 
     if (accessTokenNumUsesLimit > 0 && accessTokenNumUses > 0 && accessTokenNumUses >= accessTokenNumUsesLimit) {
       await identityAccessTokenDAL.deleteById(tokenId);
-      throw new BadRequestError({
+      throw new ForbiddenRequestError({
```

Comment:
Token it self is invalid and this isn't about a permission issue. It should be UnauthorizedError, let's doube check ones similar to these